docs/design/resource-strategy-fit-scheduling.md

```diff
@@ -0,0 +1,70 @@
+# ResourceStrategyFit Plugin
+
+## Summary
+
+The native k8s ResourceStrategyFit plug-in can only adopt one type of strategy for all resources, such as MostRequestedPriority and LeastRequestedPriority. However, in industrial practice, this design is not applicable in some scenarios. For example: in AI scenarios, we usually disperse CPU tasks in CPU machine groups to reduce hot spots. GPU tasks are gathered in GPU machine groups to reduce GPU fragmentation. Therefore, we need to expand a scheduling strategy to meet the needs of this scenario.
+
+## Motivation
+
+- Different resource types can be configured with different aggregation or dispersion strategies, and weights can be used to distinguish priorities
+
+## Design Considerationsui
+
+- The solution is more versatile, not limited to AI clusters or CPU clusters, and not limited to common CPU resources or extended GPU resources.
+
+- Different resource policies can be configured for different cluster types and prioritized in the form of weights.
+
+- Easy to expand
+
+### Goals
+
+- Different types of resources can be configured with different strategies to prioritize them in the form of weights
+
+### Non-Goals
+
+- None.
+
+## Proposal
+
+Extend one plug-ins to meet the above needs
+
+- ResourceStrategyFit
+
+## User Story
+
+### Story1
+- Users hope that different resource strategies can be adopted for different resource types. For example, in AI scenarios, they hope that pods that apply for GPU resources will occupy as many machines as possible, while pods that only apply for CPU resources will be evenly distributed to different machines.
```

Comment:
Here can give a more concrete scenario, like pytorch job? master pod using CPU should disperse to avoid hot node while worker pods using GPU should aggregate to reduce resource fragment?